### ISSUE TITLE
Add option to check for and test schemas by name

### ISSUE THREAD

**Issue by iaincollins** [glitchdigital/structured-data-testing-tool#5]
**How could the tool be better? Please describe.**
Add `--schemas <schemas>` option to test for specific schemas.

The purpose of this is to provide a way to validate schema markup to ensure it's correct according to the specifications, independent of considerations for specific implementations (e.g. irrespective of what Google considers 'valid'), so that it's more widely useful.

**Describe the solution you'd like**

This would work like the existing `---presets` option, except it would only validate against the Schema.org spec (i.e. it would not consider what search engines consider 'required' fields or acceptable values for properties).

The command would fail if the schema specified was unknown.

A test for a specified schema would generate an error if:

* The schema was not found in the input
* If a property appeared to contain an invalid value (in violation of the spec at Schema.org)

A test for a specified schema would generate a warning if:

* An object contained a property not listed as valid for the schema at Schema.org

**Comment by iaincollins** [comment]
This was partially resolved by #8 but does not support explicitly checking properties are valid.

That will require additional work which needs to be spec'd out - once that work to move it into another issue is done this issue can be closed.

**Comment by iaincollins** [comment]
Marking this issue as closed, the remainder of this work is now part of the milestone plan for version 5.

**Comment by iaincollins** [comment]
See issue #14 to track the follow up work around validating Schema properties.
